nestkernel/connection_manager.cpp

```diff
@@ -195,38 +201,40 @@ nest::ConnectionManager::set_synapse_status( index gid,
 void
 nest::ConnectionManager::delete_connections_()
 {
-  for ( tVSConnector::iterator it = connections_.begin();
-        it != connections_.end();
-        ++it )
+#ifdef _OPENMP
+#pragma omp parallel
   {
-    for ( tSConnector::nonempty_iterator iit = it->nonempty_begin();
-          iit != it->nonempty_end();
-          ++iit )
+#pragma omp for schedule( static, 1 )
```

Comment:
@tammoippen I think you added the `omp for schedule`. Do you have a comment on this?